### PR TITLE
[TT-6727] Restructured and clarified wording for Basic Auth

### PR DESCRIPTION
**Pull request by andyo-tyk**
### For internal users - please add a Jira DX PR ticket to the subject!
---
### Preview Link
https://deploy-preview-3167--tyk-docs.netlify.app/docs/nightly/basic-config-and-security/security/authentication-authorization/basic-auth/

### Description
To address TT-6727
Restructured and improved the wording to improve clarity.

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md [hunk 1]
```diff
@@ -1,21 +1,51 @@
...
   main:
     parent: "Authentication & Authorization"
     identifier: security-basic-authentication
 weight: 5 
 ---
 
-Tyk supports using basic authentication as an access key in the same way as any other access method.
+## What is Basic Authentication?
+
+Basic Authentication is a standard authentication mechanism supported by every standards-compliant HTTP server, it is also supported by almost every single web browser, which makes it an excellent access control method for smaller APIs.
```
Comment (dcs3spp) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md]:
```suggestion
Basic Authentication is a standard authentication mechanism supported by every standards-compliant HTTP server. It is also supported by almost every single web browser, which makes it an excellent access control method for smaller APIs.
```

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md [hunk 2]
```diff
@@ -1,21 +1,51 @@
...
+
+> In the context of an HTTP transaction, basic access authentication is a method for an HTTP user agent to provide a user name and password when making a request. HTTP Basic authentication implementation is the simplest technique for enforcing access controls to web resources because it doesn’t require cookies, session identifiers, or login pages; rather, HTTP Basic authentication uses standard fields in the HTTP header, obviating the need for handshakes. (Source: Wikipedia)
+
+### How does Basic Authentication work?
+
+Aa API request made using Basic Authentication will have an `Authorization` header that contains the API key.
```
Comment (dcs3spp) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md:20]: I have read the [create a basic authentication section](https://deploy-preview-3167--tyk-docs.netlify.app/docs/nightly/basic-config-and-security/security/authentication-authorization/basic-auth/#create-a-basic-authentication-user) below. 

Does the `Authorization` header contain the key or the username and password that is somehow mapped to a key?

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md [hunk 3]
```diff
@@ -1,21 +1,51 @@
...
+description: "Using Basic Authentication to lock-down your APIs with Tyk"
 menu:
   main:
     parent: "Authentication & Authorization"
     identifier: security-basic-authentication
 weight: 5 
 ---
 
-Tyk supports using basic authentication as an access key in the same way as any other access method.
+## What is Basic Authentication?
```
Comment (dcs3spp) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md:13]: Yep, I can see why you have restructured. In the current live website this section is at the end of the page.

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md [hunk 4]
```diff
@@ -27,16 +57,34 @@ To enable Basic Authentication on your API using the Tyk Dashboard:
...
+As you can see in the above example, enabling Basic Authentication is as simple as setting a flag for the feature in your API Definition object. Since Basic Authentication is a standard, Tyk will always look for the credentials as part of the `Authorization` header.
+
+## Create a Basic Authentication user
+
+When using Basic Authentication, the API key used to access the API is not generated by the Tyk system, instead you need to create at least one Basic Authentication user in the Tyk Gateway. Tyk will compare the Basic Authentication key provided in the request against the list of users you have created.
 
-And then add a username & password:
+### Using Tyk Dashboard
 
+You can use the Tyk Dashboard to create an access keywith Basic Authentication simply select the API for which you selected Basic Authentication and, in the Authentication tab, you can see that Basic Authentication settings are automatically displayed.
```
Comment (dcs3spp) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md]:
```suggestion
You can use the Tyk Dashboard to create an access key with Basic Authentication. Simply select the API for which you want to use Basic Authentication with. In the Authentication tab you should see that Basic Authentication settings are automatically displayed.
```

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md [hunk 5]
```diff
@@ -46,29 +94,11 @@ $ curl http://localhost:8080/basicauth/get \
 $ curl http://myusername:mypassword@localhost:8080/basicauth/get
 <200 response from upstream>
 ```
+We have full tutorials to guide you to [create an API Key]({{< ref "getting-started/create-api-key" >}}) via the Dashboard.
```
Comment (dcs3spp) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md:98]:
```suggestion
We have full tutorials to guide you through the process of [creating an API Key]({{< ref "getting-started/create-api-key" >}}) via the Dashboard.
```

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md [hunk 6]
```diff
@@ -1,21 +1,51 @@
...
+
+Basic Authentication is a standard authentication mechanism supported by every standards-compliant HTTP server, it is also supported by almost every single web browser, which makes it an excellent access control method for smaller APIs.
+
+> In the context of an HTTP transaction, basic access authentication is a method for an HTTP user agent to provide a user name and password when making a request. HTTP Basic authentication implementation is the simplest technique for enforcing access controls to web resources because it doesn’t require cookies, session identifiers, or login pages; rather, HTTP Basic authentication uses standard fields in the HTTP header, obviating the need for handshakes. (Source: Wikipedia)
+
+### How does Basic Authentication work?
+
+Aa API request made using Basic Authentication will have an `Authorization` header that contains the API key.
+
+The value of `Authorization` will be in the form:
```
Comment (dcs3spp) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md]:
```suggestion
The value of the `Authorization` header will be in the form:
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks for restructuring this document @andyo-tyk 

Structure looks better moving the explanation of the concepts for Basic Authentication to the beginning of the document. 

I have reviewed with minor suggestions

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md (hunk 2 above)
Comment (andyo-tyk) [reply tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md:20]: The section you linked is the method to register a key within the Gateway (i.e. to store valid credentials).
In the Authorization header, you need to provide valid credentials, which the Gateway then checks against its list.

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md (hunk 4 above)
Comment (andyo-tyk) [reply tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md]:
```suggestion
You can use the Tyk Dashboard to register a Basic Authentication key that can then be used to access your API. 

When you select the API, you can see that Basic Authentication settings are automatically displayed in the Authentication tab:
```

**Review by andyo-tyk — COMMENTED** [review]
I think this is clearer

**Review by dcs3spp — APPROVED** [review]

**Comment by dcs3spp** [comment]
@andyo-tyk I've approved PR. Once @lghiur or @furkansenharputlu complete a peer review will merge and release

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md [hunk 7]
```diff
@@ -1,21 +1,51 @@
 ---
 date: 2017-03-23T15:28:35Z
 title: Basic Authentication
-tags: ["Authentication", "Security"]
-description: "Using basic authentication to lock-down your APIs with Tyk"
+tags: ["Authentication", "Security", "Basic Auth"]
```
Comment (titpetric) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md]:
```suggestion
tags: ["Authentication", "Security", "Basic Authentication"]
```

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md [hunk 8]
```diff
@@ -1,21 +1,51 @@
...
     identifier: security-basic-authentication
 weight: 5 
 ---
 
-Tyk supports using basic authentication as an access key in the same way as any other access method.
+## What is Basic Authentication?
+
+Basic Authentication is a standard authentication mechanism supported by every standards-compliant HTTP server. It is also supported by almost every single web browser, which makes it an excellent access control method for smaller APIs.
+
+> In the context of an HTTP transaction, basic access authentication is a method for an HTTP user agent to provide a user name and password when making a request. HTTP Basic authentication implementation is the simplest technique for enforcing access controls to web resources because it doesn’t require cookies, session identifiers, or login pages; rather, HTTP Basic authentication uses standard fields in the HTTP header, obviating the need for handshakes. (Source: Wikipedia)
```
Comment (titpetric) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md]: Is this a needed citation? "Obviating" is making me pick up a thesaurus

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md [hunk 9]
```diff
@@ -1,21 +1,51 @@
...
   main:
     parent: "Authentication & Authorization"
     identifier: security-basic-authentication
 weight: 5 
 ---
 
-Tyk supports using basic authentication as an access key in the same way as any other access method.
+## What is Basic Authentication?
+
+Basic Authentication is a standard authentication mechanism supported by every standards-compliant HTTP server. It is also supported by almost every single web browser, which makes it an excellent access control method for smaller APIs.
```
Comment (titpetric) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md]:
```suggestion
Basic Authentication is a standard authentication mechanism implemented by HTTP servers, clients and web browsers. This makes it an excellent access control method for smaller APIs.
```

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md (hunk 8 above)
Comment (andyo-tyk) [reply tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md]: Agreed - I was trying to make minimal content change while improving clarity, but I didn't see the point of this quote. Happy to remove it.

**Review by lghiur — APPROVED** [review]

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md (hunk 8 above)
Comment (dcs3spp) [reply tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md]: Will merge once comment resolved.

Comment (andyo-tyk) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/basic-auth.md]:
```suggestion
```

**Review by andyo-tyk — COMMENTED** [review]
Implementing suggestions from Tit's review

**Pull request merged**